user-interface/src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Navbar from './NavBar';
+import Section1 from './Section1';
+import Section2 from './Section2';
+import Section3 from './Section3';
+import Section4 from './Section4';
+import Footer from './Footer';
+
+
+const App = () => {
+    return (
+        <div>
+            <Navbar />
+            <Section1 />
+            <Section2 />
+            <Section3 />
+            <Section4 />
+            <Footer />
+        </div>
+    );
```

Comment:
Currently, it seems like you're not using either `.jsx` or React components. I'd highly recommend using these both since they will make it easier to break up the frontend of your app into logical pieces, and will make it simpler to add any Javascript you need.

See:
- https://reactjs.org/docs/introducing-jsx.html
- https://reactjs.org/docs/components-and-props.html

Reply:
I've never used Typescript before, but I can do more research on it!

I'm confused because I though that I was using React components. I thought the code sample you commented on was React Components. Is there something I'm not understanding?

Reply:
Oh, woops. You're right that these are React components. Sorry, I'm just used to seeing ES6 classes instead of functions, but this is a perfectly fine way of doing it.

Reply:
The tutorials that I've been using to learn have been using .js files for React code. I looked it up online and found some discussion on the subject. [Here ](https://stackoverflow.com/questions/46169472/reactjs-js-vs-jsx#:~:text=So%20you%20are%20forced%20to,They're%20completely%20interchangeable!&text=Hence%20all%20React%20files%20that%20contain%20them%20are%20JSX%20and%20not%20JS.)and [here](https://github.com/airbnb/javascript/pull/985).

Based off of those 2 resources I'm not sure which one to use. What do you think?

Reply:
Haha wow, I had no idea there was such a debate regarding this. I think for this type of project, it's fine to use just `.ts` or `.js`. It's ultimately up to you, as it's a stylistic thing.

Reply:
Let me know if there's anything else I can do to improve my code!